### ISSUE TITLE
Reopening the app breaks the layout.

### ISSUE THREAD

**Issue by F1nnM** [futo-org/grayjay-android#262]
Steps to reproduce:

1. Start video in grayjay.
2. Do NOT swipe down on the video (as if you were trying to minimize it) -> This prevents the bug from occuring
3. Pause video.
4. Minimize grayjay.
5. Close picture-in-picture player -> Important for this bug
6. Open Grayjay.

Result:
![image](https://github.com/futo-org/grayjay-android/assets/12767947/7194eb95-c5b8-43f5-8ceb-59ed5d04e105)

Then, pulling down on anything but the video results in this:
![image](https://github.com/futo-org/grayjay-android/assets/12767947/526a8853-2e02-4ad9-95f7-5650ab5493f6)

Pulling down on the video fixes it.

Somewhen, when trying to reproduce it, I simply clicked on a video and ended up with this, but couldn't reproduce it..
![image](https://github.com/futo-org/grayjay-android/assets/12767947/af86daa0-9fab-452d-9898-d25350bfffc2)

**Comment by gudenau** [comment]
I'm having some similar issues with the layout breaking, I just have no idea what the cause might be. Normally when it happens the video is extremely small, about the size it would be in the small PIP. Normally it's fine in fullscreen though.

Today I had fullscreen completely break and had to restart the application to fix it. The only area that responded to touch inputs was the video itself. I had opened this video from the search by using the URL, paused it and queded a download. When I came back I dismissed the video and started playing it from my downloads only to have it break the layout.

The only common thing I can find with the layout issues is having the application in the background, other than that I have no idea.

![Screenshot_20240114-125553](https://github.com/futo-org/grayjay-android/assets/5642329/a9686c6c-2f56-48f1-8f74-eb6b6938c3c5)
![Screenshot_20240114-125605](https://github.com/futo-org/grayjay-android/assets/5642329/779c5a81-2ecf-457c-bac6-87ef85325ace)
